deltabar/deltabar_lib/deltabar_ui.py

```diff
@@ -0,0 +1,316 @@
+import ac
+import math
+import sys
+
+from deltabar_lib import colors
+from deltabar_lib import config
+
+
+class DeltaBarUI:
+  """
+  User Interface of Delta Bar application.
+  """
+
+  BACKGROUND_COLOR_RGB = colors.rgb_from_hex('#303030')
+  BACKGROUND_COLOR_ALPHA = 0.65
+  BACKGROUND_COLOR_RGBA = colors.rgba_from_rgb_alpha(BACKGROUND_COLOR_RGB, BACKGROUND_COLOR_ALPHA)
+
+  FAST_COLOR = (0.1, 1.0, 0.1, 1.0)
+  SLOW_COLOR = (1.0, 0.8, 0.0, 1.0)
+
+  def __init__(self, deltabar_data, label_tracker):
+    self.data = deltabar_data
+    self.label_tracker = label_tracker
+
+  def initialize(self):
```

Comment:
I'd recommend having all of this in reinitialize(), and remove the 'initialize' function completely.